notes/2024-02-06 biglep feedback for PR #65.md

```diff
@@ -0,0 +1,99 @@
+
+# Somewhere between general and IPLD-specific
+These are at least things that apply to the 2024Q1 cleanup for ipld, libp2p, and ipfs orgs. Some of them can be generalized.
+
+## Archived repos cause clutter
+These .yaml files can be unwieldy.  One thing I think could help would be to have a view on all the repos that aren't archived.  Some ideas (I assume from least to most amount of work):
+* Put all the archived repos at the bottom of the file.  (I still don't like this one as much because archived repos will show up when doing text search for "team X" when trying to answer "what teams does X have permissions to").
+* Have an optional separate .yaml that merges in at build/process time.  This would allow a dedicated separate file (e.g., archived_repos.yaml).
+  * Generalizing this, maybe github-mgmt can be configured to do a merge of any specified files, which lets repo organizers decide if/how they want to break up their yaml file in general?
+* Go the IPFS route and actually have an "ipfs-inactive" org.  Archived projects can get moved there.  This makes things very clear about the maintenance status and also solves the declutter problem. 
+
+## Strip out access permissions to archived repos
+It adds clutter having individuals and teams showing up with access permissions in archived repos.  
+If a repo is archived, I think we should strip permissions.  
+Someone can always unarchive and add permissions through github-mgmt.  
+In addition to this reducing clutter, I this proposed process is good because it gives clear visibility to a significant repo event (e.g., unarchiving).
+
+## Remove the w3dt-stewards team
+That is legacy both in terms of name and function.
+It was used when there was a small team who was charged with looking after the whole github org and they didn't have github-mgmt at their disposal.
+In PLv10 / PL Innovation Network era, there is no corrolary for this.
+If broad action is needed, it can be done broadly via github-mgmt.  And where it can't be done directly with github-mgmt, github-mgmt can at least be done to give broad permissions to go do the broad action.
+
+## Default access of the ipdx team
+It looks like the ipdx team has admin permissions on most repos.   
+Should we maybe reduce this and instead ensure they just have github-mgmt admin access, which still enables them to "break glass" and escalate their access when needed?
+
+## Commit/PR order
+If some of the feedback above is incorporated, I'm wondering if we should do commits or PRs in this order to help make the consequential changes more clear.
+1. Remove access permissions of users/teams to archived reports.
+2. Factor out archived repos.
+3. Remove teams like w3dt-stewards
+4. Reduce ipdx access
+5. (Anything related to renaming team names)
+6. Other changes that the IPDX scripts picks up based on audit log inactivity and based on additional manual changes.  (This is the diff we need people to be scrutinizing.)
+
+# IPLD specifics
+## Org admins
```

Comment:
2024-02-08 conversation: This is the higher-order bit and requires human intervention.  It can be done ahead of everything else and already makes things significantly more locked down.  @BigLep will take on opening PRs about reducing org owners ("members.admin") for ipfs/libp2p/ipld/etc. later 2024-02-09.

Reply:
This has been called out as the key item in the overarching tracking item in https://github.com/ipfs/ipfs/issues/511